src/index.test.ts

```diff
@@ -1,7 +1,180 @@
+import { People, copyPeopleArray, addAge, removeAge, removeAgeWithDestructure, removeAgeSimplifiedReturn, changeNameAlias, arrayOfStringNames } from "./index";
+
 //write tests here
 
-describe('Tests will go here!', () => {
-  it('should pass', () => {
-    expect(true).toBeTruthy()
-  })
-})
\ No newline at end of file
+
+describe('copy people array', () => {
```

Comment:
in your describe blocks,  make sure to specify the function being tested

```suggestion
describe('copyPeopleArrary', () => {
```